apps/api/src/app/vercel-integration/vercel-integration.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { USE_CASES } from './usecases';
+import { VercelIntegrationController } from './vercel-integration.controller';
+import { SharedModule } from '../shared/shared.module';
+import { HttpModule } from '@nestjs/axios';
```

Comment:
This should be above the relative imports

Reply:
hm @scopsy would be great if we could auto fix this if we think this should be a pattern we follow 😄

Reply:
I think there's one eslint rule for this, we can add it